app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+
+    if (
+            len(command_list) != 3
+            or command_list[0] != "cp"
+            or command_list[1][-3:] != "txt"
+            or command_list[2][-3:] != "txt"
+    ):
+        print("The command must be in the following format:")
+        print("cp file.txt file-copy.txt")
+        return
+
+    source_name = command_list[1]
+    copy_name = command_list[2]
```

Comment:
what are command_list[0], command_list[1], command_list[2]?
create separated variables for them

Reply:
Fixed